### DISCUSSION TITLE
Mobile app not loading issues for months

### DISCUSSION THREAD

**Discussion by supermarin** [community/community#121572] [Mobile] [answered]
Whenever I tap on a GitHub link from email, or just try to browse through the mobile app, the issue details page never loads. It just spins forever.
I tried deleting and reinstalling the app multiple times, but the issue persists. I can't use GitHub on mobile at all if I have this app installed.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.159.0 (129045654)
Device: iPhone 15 Pro (iOS 17.4.1)
</details>

**Comment by stevepopovich** [comment]
Hi @supermarin sorry to hear this is happening. Are other parts of the app loading? Do you have a video of what is happening?

**Reply by supermarin** [reply]
https://github.com/community/community/assets/627285/ae46b19b-9ad9-4b67-a3b3-1869debc1fab

**Reply by stevepopovich** [reply, marked as answer]
Ok, sorry for the delay @supermarin we see you have a possible "scope" issue. Could you try manually **logging out** of the app, not reinstalling. 
- Long click your profile picture
- Edit
- Logout
- Log back in

Let me know if that helps!

**Reply by supermarin** [reply]
Thank you @stevepopovich that resolved it
